### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by TemesgenA**
## Mobile Version
Knowledge implemented:
- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements on the page.
- Use CSS Grid to place elements on the page.
- Build a personal portfolio site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,37 +7,194 @@
...
+                        don’t hestiate to contact me.</p>
+                </article>
+                <h4>Lets connect</h4>
+                <ul>
+                    <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-maxcdn" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-github" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-hand-peace-o" aria-hidden="true"></i></li>
+                </ul>
```
Comment (micheaol) [inline index.html]: Please ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has `a link (`a` tag)` according to the project requirement.

File: index.html [hunk 2]
```diff
@@ -7,37 +7,194 @@
...
+                        feature or website Look through some of my work and experience!
+                        If you like what you see and have a project you need coded,
+                        don’t hestiate to contact me.</p>
+                </article>
+                <h4>Lets connect</h4>
+                <ul>
+                    <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-maxcdn" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-github" aria-hidden="true"></i></li>
```
Comment (micheaol) [inline index.html]: Please ensure that each section has the margins (top, bottom, left, right) defined in the design, and the font and colors (background and font) defined for the template according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @TemesgenA , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by m05e5 — APPROVED** [review]
# Hi @TemesgenA , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**